test/service/shortcut_event/shortcut_event_test.dart

```diff
@@ -209,15 +234,56 @@ void main() async {
           );
         }
       }
+
       if (Platform.isWindows || Platform.isMacOS || Platform.isLinux) {
         await editor.pressLogicKey(
           key: LogicalKeyboardKey.end,
         );
       }
+
       expect(
         editor.documentSelection,
         Selection.single(path: [1], startOffset: text.length),
       );
     });
+
+    testWidgets('delete sentence to beginning', (tester) async {
+      const text = "Hello World!";
+      final editor = tester.editor
+        ..insertTextNode(text)
+        ..insertTextNode(text);
+
+      await editor.startTesting();
+
+      await editor.updateSelection(
+        Selection.collapsed(Position(path: [1], offset: 10)),
+      );
+
+      expect(editor.documentLength, 2);
+
+      if (Platform.isMacOS) {
+        await editor.pressLogicKey(
+          key: LogicalKeyboardKey.backspace,
+          isMetaPressed: true,
+        );
+      } else {
+        await editor.pressLogicKey(
+          key: LogicalKeyboardKey.backspace,
+          isControlPressed: true,
+          isAltPressed: true,
+        );
+      }
```

Comment:
```suggestion
      await editor.pressLogicKey(
        key: LogicalKeyboardKey.backspace,
        isMetaPressed: Platform.isMacOS,
        isControlPressed: !Platform.isMacOS,
        isAltPressed: !Platform.isMacOS,
      );
```We can do this everywhere else where applicable as well.